server/api/cli/cli.controller.js

```diff
@@ -39,17 +39,22 @@ exports.run = function (req, res) {
   dirName += dateTime.getFullYear()
   const execFile = req.body.fileName.replace('.c', '')
   const cmd = `"${dirName}/${execFile}"`
-  exec(cmd, { timeout: 10000 }, // Process will time out if running for > 10 seconds.
+  var cp = exec(cmd, { timeout: 10000 }, // Process will time out if running for > 10 seconds.
     (error, stdout, stderr) => {
       if (error) {
-        return res.send(200, error)
+        return res.send(200, {"info":error, "process":cp.pid});
       }
       if (error !== null) {
       }
-      return res.send(200, stdout)
+      return res.send(200, {"info":stdout, "process":cp.pid});
     })
 }
 
+exports.killProcess = function(req, res) {
+	var result = process.kill(req.body.process);
+	return res.send(200, result);
```

Comment:
Please use `res.status(200).send(result)` as `res.send(200, result)` is deprecated in new versions of express.